plugins/modules/nd_rest.py

```diff
@@ -48,6 +48,11 @@
     - The file path containing the body of the HTTP request.
     type: path
     aliases: [ config_file ]
+  ignore_previous_state:
```

Comment:
Isn't this the same as https://github.com/CiscoDevNet/ansible-aci/blob/master/docs/optimizing.md#suppress_previous we implemented in ACI? I would align the naming if it is.

Reply:
Yes, this attribute does the same as suppress_previous, but introduced to avoid unsupported GET for certain ND API endpoints that is "/api/config/route".